### ISSUE TITLE
[Playground- Lines]: Ensures every form element has a label.[Sev2][WCAG1.3.1][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#10]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to the ‘Playground’ link and activate it.
4) Navigate to 'Lines' tab and activate it.
5) Now run Accessibility Insights for web tool and observe the issue.

**Actual:**
Label is not associated with 'Lines' tab in 'Playground' link.
 
**Expected:**
Ensure every form element must have a label attribute in the page.

**User Impact:**
Labels are used to provide necessary information to the user to ensure that the component’s purpose is clear. In the absence of label information then user who are relying on screen reader may not be able to get the guidance on how to fill out the form edit field so the end users will get confused.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
<img width="960" alt="Ensures every form element has a label_Lines" src="https://user-images.githubusercontent.com/85154417/121009536-eaf51080-c7b1-11eb-9960-a749d75b88ea.png">

**Comment by apostolisms** [comment]
[This](https://github.com/microsoft/tsdoc/pull/303) PR fixes the issue.
